index.html

```diff
@@ -1,405 +1,347 @@
-            <img src="icons&imgs/Group55.png" alt="arrow icon" />
-            <p class="work">Work</p>
-          </div>
+          <li>
+            <a class="medium flex" role="button" href=""><img class="comunic" src="icons&imgs/Vector.png"
+                alt="medium icon" /></a>
+          </li>
+        </ul>
+        <div class="elementRight-2 flex">
+          <img src="icons&imgs/Group55.png" alt="arrow icon" />
+          <p class="work">Work</p>
         </div>
       </div>
+    </div>
 
-      <div id="portfolio" class="sub-container-2 flex column">
-        <section class="recentWork flex column">
-          <div class="hdr flex column">
-            <h2 class="mainTxt-2">MY RECENT WORK</h2>
-            <hr class="divider" />
-          </div>
-          <div class="workBlocks">
-            <div class="featuredImg flex column">
-              <img src="_blank" alt="" class="imgPlaceHolder one" />
-              <div class="rightBlock two">
-                <h3 class="titlePost">Multi-Post Stories</h3>
-                <p class="supportingTxt">
-                  A daily selection of privately personalized reads; no accounts
-                  or sign-ups required. has been the industry's standard dummy
-                  text ever since the 1500s, when an unknown printer took a
-                  standard dummy text.
-                </p>
-                <ul class="tags flex">
-                  <li>css</li>
-                  <li>html</li>
-                  <li>bootstrap</li>
-                  <li>Ruby</li>
-                </ul>
-                <button type="button" disabled class="button normal preview">
-                  See project
-                </button>
-              </div>
-            </div>
-            <div class="featuredImg flex column">
-              <img src="_blank" alt="" class="imgPlaceHolder two" />
-              <div class="rightBlock one">
-                <h3 class="titlePost">Multi-Post Stories</h3>
-                <p class="supportingTxt">
-                  A daily selection of privately personalized reads; no accounts
-                  or sign-ups required. has been the industry's standard dummy
-                  text ever since the 1500s, when an unknown printer took a
-                  standard dummy text.
-                </p>
-                <ul class="tags flex">
-                  <li>css</li>
-                  <li>html</li>
-                  <li>bootstrap</li>
-                  <li>Ruby</li>
-                </ul>
-                <button type="button" class="button normal preview" disabled>
-                  See project
-                </button>
-              </div>
+    <div id="portfolio" class="sub-container-2 flex column">
+      <section class="recentWork flex column">
+        <div class="hdr flex column">
+          <h2 class="mainTxt-2">MY RECENT WORK</h2>
+          <hr class="divider" />
+        </div>
+        <div class="workBlocks">
```

Comment:
- [ ] Great job adding your project modals dynamically, however you also need to add your project cards themselves dynamically as you did for the project modal.
- [ ] I kindly suggest that you remove the project cards you added manually in your HTML and add them dynamically through JavaScript as you did for the modal

Reply:
This was week2 assignment, and I don't really have any projects to show yet. However the project requirement is to do the pop up modal only as it is mentioned in the title.